packages/frontend/src/components/stages/StageTooltip.tsx

```diff
@@ -1,34 +1,83 @@
-import { StageConfig } from '@l2beat/config'
+import { Stage, StageConfig } from '@l2beat/config'
 import React from 'react'
 
+import { MissingIcon } from '../icons'
 import { StageBadge } from './StageBadge'
+import { StageDisclaimer } from './StageDisclaimer'
 
 export interface StageTooltipProps {
   item?: StageConfig
 }
 
 export function StageTooltip({ item }: StageTooltipProps) {
   if (!item) return null
+
   return (
-    <div className="w-88 flex flex-col gap-4">
-      <span className="font-bold">
-        <StageBadge stage={item.stage} />
+    <div className="flex max-w-[300px] flex-col gap-4 py-1">
+      <span>
+        <StageBadge stage={item.stage} className="font-medium" />
+        <span className="ml-2 inline-block font-medium">
+          {getStageName(item.stage)}
+        </span>
       </span>
-      <hr className="border-gray-650" />
-      {item.missing && (
-        <div>
-          <p className="mb-2 text-[13px] uppercase leading-tight text-gray-50">
-            Items missing for {item.missing.nextStage}:
-          </p>
-          <ul className="ml-2 list-none space-y-2">
-            {item.missing.requirements.map((requirement, i) => (
-              <li className="w-60" key={i}>
-                ❌ {requirement}
-              </li>
-            ))}
-          </ul>
-        </div>
+      {item.stage === 'UnderReview' ? (
+        <>
+          Projects under review might present uncompleted information & data.
```

Comment:
this text seems a bit too long, but it is a nitpick

Reply:
What do you propose then?

Reply:
Let's leave it for now, I do not have an idea